boards/arm/google_dragonclaw/google_dragonclaw.dts

```diff
@@ -92,12 +92,37 @@
 };
 
 /*
- * The board uses STM32F412CG in UFQFPN48 package in which gpio[c-h] is not
+ * Set flags of unused pins as GPIO_ACTIVE_HIGH (0 << 0), which will be
```

Comment:
maybe mention: without setting the gpio as either input or output, that's the important bit

Reply:
IMO the `GPIO_DISCONNECTED` part implicates that isn't input or output, but added anyway if you think it can help to understand why it is done this way.

Reply:
Thanks, yeah I mean maybe it's a "me" problem but I would expect the default to be "input", there's nothing that says "disconnected" here and this is a STM32 specific combination, I think it's worth being clear.